Lib/test/typed_dict.py

```diff
@@ -0,0 +1,8 @@
+from __future__ import annotations
```

Comment:
Please add a comment/docstring at the top describing the purpose of this file for future generations.